### ISSUE TITLE
Complexity Measures

### ISSUE THREAD

**Issue by devoworm** [devoworm/Group-Meetings#3]

**Comment by jesparent** [comment]
following.

**Comment by jesparent** [comment]
There is definitely overlap between this and my FrontierMap project

**Comment by jesparent** [comment]
This has been well on the back burner, and I'm not sure if it will get life soon, but, when it does I'd be interested in it. I think this may be one facet to swing -around to wrt Dev-AI as well? @devoworm  @ OREL

**Comment by Orthogonal-Research-Lab** [comment]
Yes, we might get further with a tie-in to dev-AI. The Math of DevoWorm stuff might also fit into this.
